pkgs/applications/editors/vim/configurable.nix

```diff
@@ -92,26 +95,9 @@ composableDerivation {
 
       // edf {
         name = "python";
-        feat = "pythoninterp";
+        feat = "python${pythonVersion}interp";
         enable = {
-          nativeBuildInputs = [ python ];
-        } // lib.optionalAttrs stdenv.isDarwin {
```

Comment:
This just removes the `--with-python-config-dir` configuration flag, which is not needed, as it pulls in the correct location during the configuration process.